### ISSUE TITLE
[Feature Request] Character mapper and Dictionary support For AI translation

### ISSUE THREAD

**Issue by mcabel** [libretro/RetroArch#9761]
Sometimes, games have weird fonts that may not be picked up correctly by OCR's. In such case, one could have an interface that Shows game characters and mark it as blank by default when never seen or when not recognized. But users can open a menu to see the characters, and fill them up manually, and TELL the system the real interpretation of the character.

Similar to what the emulator Mesen desktop version can to when text hooking.(screenshot example shown below).

 Also, it would be nice for the system to read up to n Dictionary files, for offline based translation. Traditionally LEC, Vicon and ATLAS dictionary files are very popular in translation utilities like VN Reader(used to translate visual novels with machine translation). Also some user-made dictionary (i.e. .txt )format that may read from m amount of files, would be nice to support the bigger dictionaries like LEC.

 
![image](https://user-images.githubusercontent.com/10392829/69294512-1c5eb900-0bd9-11ea-805e-9479c156b379.png)

**Comment by kira1000** [comment]
I agree, if we could use offline dictionary files, it would mean that larger JRPGs/VNs could be played without worrying about any sort of 10k character limit each month.
